src/state/effects/types.ts

```diff
@@ -1,11 +1,24 @@
 import { Round } from '../round';
 import { EFFECT_HIDDEN } from '../../constants';
 
+export const GREMLIN_CHANCE_KEY = 'gremlinChance';
+export const CAPACITY_CHANGE_KEY = 'capacityChange';
+export const USER_STORY_CHANCE_KEY = 'userStoryChance';
+
+type GremlinChanceEffectProps = {
+  [GREMLIN_CHANCE_KEY]: number;
+  [CAPACITY_CHANGE_KEY]?: never;
+  [USER_STORY_CHANCE_KEY]?: never;
+};
 type CapacityEffectProps = {
-  capacityChange: number;
+  [GREMLIN_CHANCE_KEY]?: never;
+  [CAPACITY_CHANGE_KEY]: number;
+  [USER_STORY_CHANCE_KEY]?: never;
 };
 type UserStoryChanceEffectProps = {
-  userStoryChance: number;
+  [CAPACITY_CHANGE_KEY]?: never;
+  [GREMLIN_CHANCE_KEY]?: never;
+  [USER_STORY_CHANCE_KEY]: number;
 };
```

Comment:
the above changes make it so that an effect can only change one thing. As stated above, this is to prevent filtering bugs if something should cause two effects, it should return an array of effects